lte/cloud/helm/lte-orc8r/Chart.yaml

```diff
@@ -13,7 +13,7 @@ apiVersion: v2
 appVersion: "1.0"
 description: A Helm chart for magma orchestrator's lte module
 name: lte-orc8r
-version: 0.2.5
+version: 0.2.6
```

Comment:
No need to bump the helm chart version -- we didn't change anything that would require redeploying/upgrading the charts, so all we would need to do is upgrade the container image version

Side note, this change is meant to be backwards-compatible (up to the default page size)